modules/monitoring-alertmanager-default-configuration.adoc

```diff
@@ -28,4 +28,6 @@ receivers:
 
 {product-title} monitoring ships with the Watchdog alert that always triggers by default to ensure the availability of the monitoring infrastructure.
```

Comment:
this sentence seems redundant with the new section below.

Reply:
ACK, I've merged the first sentence and the first sentence of the second paragraph.